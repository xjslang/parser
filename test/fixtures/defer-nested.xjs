function outer() {
  console.log('outer start')
  
  function inner() {
    console.log('inner start')
    defer {
      console.log('inner defer')
    }
    console.log('inner end')
  }
  
  inner()
  console.log('outer end')
}
outer()
