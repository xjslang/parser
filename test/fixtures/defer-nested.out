outer start
inner start
inner end
inner defer
outer end
